.gitignore

```diff
@@ -45,7 +45,9 @@ __pycache__/
 # Distribution / packaging
 .Python
 env/
-build/
+build/*
+!build/.keep
```

Comment:
What are these `.keep` directories for? Here and below

Are they specific to your workflow or is it something that is expected to exist for all users of Harmony?